run-oft.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# [impl->req~run-oft-trace-command~1]
+
+report_file_name=$1
+report_format=$2
+files=$3
+
+echo "::notice::using OpenFastTrace JARs from: ${LIB_DIR}"
+
+# Run OpenFastTrace
+if (java -cp "${LIB_DIR}/*" \
+  org.itsallcode.openfasttrace.core.cli.CliStarter trace -o "${report_format}" \
```

Comment:
Would it be possible to start OFT with `java -jar ...`? This way we don't need to hard-code the main class name.

Reply:
I don't think so. From the Java documentation:
> -jar jarfile
Executes a program encapsulated in a JAR file. The jarfile argument is the name of a JAR file with a manifest that contains a line in the form Main-Class:classname that defines the class with the public static void main(String[] args) method that serves as your application's starting point. When you use -jar, the specified JAR file is the source of all user classes, and other class path settings are ignored. If you're using JAR files, then see jar.

Reply:
OK, you are right, I forgot that we need to add the plugin to the classpath.
I will try to put the plugin into OFT's search path, then `-jar` should work. But this is something for another PR: